macos/dynamic_color.podspec

```diff
@@ -1,5 +1,10 @@
 Pod::Spec.new do |s|
   s.name             = 'dynamic_color'
+  s.authors          = 'Ebrahim Byagowi'
```

Comment:
Should I put Google's name here or myself or both...?

Reply:
Material Flutter team + your name

Reply:
Done